samples/nrf9160/agps/src/main.c

```diff
@@ -26,9 +31,45 @@ static uint64_t start_search_timestamp;
 static uint64_t fix_timestamp;
 static struct k_delayed_work gps_start_work;
 static struct k_delayed_work reboot_work;
+#if defined(CONFIG_NRF_CLOUD_PGPS)
+static struct k_delayed_work manage_pgps_work;
```

Comment:
This still needs to be converted :slightly_smiling_face:

Reply:
Ah, I see.  I hadn't noticed that.